### DISCUSSION TITLE
Will you add potato GLaDOS?

### DISCUSSION THREAD

**Discussion by CN1o2** [BEEmod/BEE2-items#4184] [Tech Support / Q&A]

**Comment by Konclan** [comment]
Remember to read the [contributing guidelines](https://github.com/BEEmod/.github/blob/c865feea6f7f4bd9a65e8aaecac9c7731db9628f/contributing.md#this-is-not-a-forum) before posting. For questions please either head over to the [BEEmod item repositories' discussion page](https://github.com/BEEmod/BEE2-items/discussions) or go to our [Discord Server](https://discord.gg/EvC8Fku)
